README.md

```diff
@@ -50,11 +50,12 @@ Note that a systemd `.service` is also installed. You will probably need to modi
 
 ### Environment Variables
 - `TESSERACT_DATABASE_URL`: required, is the address of the database; make sure to include the user, password, and database name.
-- `TESSERACT_SCHEMA_FILEPATH`: required, should point to the location on disk for the tesseract schema file.
-- `TESSERACT_LOGIC_LAYER_CONFIG_FILEPATH`: optional, should point to the location on path for the logic layer configuration.
+- `TESSERACT_DEBUG`: boolean, `true` is a debug mode for data/calculations. As of now, the only feature is the `rca` calculation will show `a b c d` sub-aggregations for the calculation`(a/b)/(c/d)`.
```

Comment:
I think while we're at it we should update this text. I believe now setting this to true also returns mote detailed error messages in the response when queries fail, so it goes beyond just RCA.